test/functional/feature_asset_locks.py

```diff
@@ -500,5 +500,25 @@ def run_test(self):
         assert_equal(new_total, self.get_credit_pool_balance())
         self.check_mempool_size()
 
+        self.activate_mn_rr(expected_activation_height=3090)
+        self.log.info(f'height: {node.getblock(node.getbestblockhash())["height"]} credit: {self.get_credit_pool_balance()}')
+        reward = 2299859813
```

Comment:
@knst Can you just explain how this value is calculated?

Reply:
Sure. I just took a reward that should be on this height for masternodes and multiplied it to 0.375: `trunc(6132959502  * 0.375) =  2299859813`

The amount of dash that should be generated by this block is: `6132959502`.
This amount is already used in [unit test, here](https://github.com/dashpay/dash/blob/master/src/test/block_reward_reallocation_tests.cpp#L327):
```
// Reward split should reach ~60/40 after reallocation is done
...
src/test/block_reward_reallocation_tests.cpp:        BOOST_CHECK_EQUAL(pblocktemplate->voutMasternodePayments[0].nValue, 6132959502); // 0.6
```

@ogabrielides , should I put this explanation in the script or this comment in PR is enough?

Reply:
@knst No it is fine. Thanks!

Reply:
let's replace it with actual calculations maybe abff1f038 (and also test `getblocktemplate` as a side effect)